src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            return (
+              <li
+                key={tab.id}
+                className={cn({ 'is-active': selectedTab.id === tab.id })}
```

Comment:
```suggestion
                className={cn({ 'is-active': isSelected })}
```
You can use a variable to check it and reuse it further